src/DFC_BusinessOntology.owl

```diff
@@ -10264,6 +10948,16 @@ N.B. There is an expectation that either the ProductionFlow or the ConsumptionFl
         <IRI>#transforms</IRI>
         <Literal xml:lang="en">transforms</Literal>
     </AnnotationAssertion>
+    <AnnotationAssertion>
+        <AnnotationProperty abbreviatedIRI="rdfs:label"/>
+        <IRI>#transportedBy</IRI>
+        <Literal xml:lang="fr">est transporté par</Literal>
+    </AnnotationAssertion>
+    <AnnotationAssertion>
+        <AnnotationProperty abbreviatedIRI="rdfs:label"/>
+        <IRI>#transportedBy</IRI>
+        <Literal xml:lang="en">transported by</Literal>
```

Comment:
"The Shipment that transports Stock"

Reply:
We're missing labels as well as a comment for `#transports`

Suggest:
"The Stock that is transported by a Shipment."